src/pages/BlockList/index.tsx

```diff
@@ -222,6 +223,10 @@ export default () => {
         )}
         <div className="block_list__pagination">
           <Pagination currentPage={currentPage} totalPages={totalPages} onChange={setPage} />
+          <a href="/export-transactions?format=block" target="_blank">
```

Comment:
It would be better to use `Link` component, which as I recall allows the react-router to be more aware of changes.